### ISSUE TITLE
Suggestion: Multiple cells selection

### ISSUE THREAD

**Issue by LeonardoGentile** [HappenApps/Quiver#38]
Sometimes I move multiple cells from one note to another. Right now I do so by `cmd-shift-x` then `cmd-shift-v` in the new note. When I have more than one cell to cut-paste this could become a long task and I also should be very careful about the paste order.

It would be better to select multiple cells at once (`cmd-click` for example) then apply the operations to all of them (multiple cut then multiple paste in this case)

**Comment by ylian** [comment]
Great idea. I have struggled to find a good UI for multi-cell operations, but `cmd-click` seems to be a great option.

**Comment by LeonardoGentile** [comment]
A problem with `cmd-click` could be that if triggered within a code cell for more than one time will select more lines (very useful by the way). So it might not be clear if we want to select multiple lines within a code cell or select multiple cells..

**Comment by ylian** [comment]
That's true. It's the same shortcut used for multi-cursor in a code cell.

**Comment by LeonardoGentile** [comment]
OT: by the way multi-cursor in text cell would be also interesting. If feasible I'll open another ticket

**Comment by ylian** [comment]
I agree but unfortunately that is not feasible. Text cells use HTML contenteditable, other cells use Ace. It will be a huge amount of work to implement that in a HTML contenteditable. We should thank Ace indeed.

**Comment by LeonardoGentile** [comment]
Indeed, thanks ace and thank you, no problem :+1:

**Comment by LeonardoGentile** [comment]
Any news on this? 
Also would be useful a `select all cell` command for massive copying-pasting of cells

**Comment by ylian** [comment]
Not in this release.

**Comment by brantfaircloth** [comment]
First, Quiver is really excellent - thank you @ylian.

+1 for copying/cutting and pasting multiple cells to a new location.  I use Quiver to track data processing steps, and many of these steps are mostly redundant with slight modifications - so being able to copy and paste many cells would be quite helpful.

**Comment by eanhuddleston** [comment]
Another +1 for being able to select multiple cells at once (for copying/pasting, reordering multiple cells at once within a note, etc.).

**Comment by ghost** [comment]
Hello, is there any update on this?  I find myself running into this problem often.

**Comment by derick** [comment]
Would love to be able to do this.  So frustrating when this happens in an otherwise killer app.

**Comment by lhrkkk** [comment]
That's the last feature which quiver needed to be a killer app.

**Comment by chodorowicz** [comment]
Yeah, this feature would be mucho appreciated 🙇

**Comment by brettz9** [comment]
Yes, this multiple cell selection (for copy-paste as well as cutting) feature (along with #170 on wiki internal/external link shorthand) would make this app a 5 star app without hesitation...

For the idea for "select-all cells", I think command-shift-A would be a handy default (incidentally, I think command-return is a better default for new cells--though I've already tweaked this for myself in `System Preferences->Keyboard->Shortcuts->App Shortcuts`)...

One other idea that would be useful, but admittedly not as critical, would be merging selected notes (selecting by command-click as well and then right-clicking to merge)

**Comment by kimonoki** [comment]
Need this feature. I end up using only one markdown cell to make things easier.

**Comment by gandalfsaxe** [comment]
Being able to select and move multiple cells is much needed 👍

**Comment by luispuerto** [comment]
I'd also like to have this feature.

**Comment by notoriousturtle** [comment]
Also.

**Comment by benloftus** [comment]
+1

**Comment by susansan** [comment]
+1

**Comment by tomkyle** [comment]
+1

**Comment by Sanabria** [comment]
+1

**Comment by reiy-leo** [comment]
any progress?

**Comment by nadrino** [comment]
Same question, it's been nearly 5 years the suggestion has been raised. Hope the project hasn't been discontinued

**Comment by franciskolms** [comment]
Can this be done for at least only the text cells or cells that are of the same type? This basic 'select all' feature is a must have for notes that grow in size for various reasons.

**Comment by charlottewright** [comment]
I'd also think this feature would be great.

**Comment by prestonjcrowe** [comment]
+1
